src/main/js/component/providers/BlackDuckConfiguration.js

```diff
@@ -88,9 +89,14 @@ class BlackDuckConfiguration extends React.Component {
 
                 <form className="form-horizontal" onSubmit={this.handleSubmit}>
                     <div>
-                        <ReadOnlyField label="Url" name="blackDuckUrl" readOnly="true" value={this.props.blackDuckUrl}/>
+                        <TextInput label="Url"
```

Comment:
I'm worried that allowing people to change config while Alert is running will cause some issues.